sdk/core/Azure.Core.Amqp/src/AmqpAnnotatedMessage.cs

```diff
@@ -161,7 +161,7 @@ section switch
         /// <seealso href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#section-message-format"/>
         /// </summary>
         /// <returns>A <see cref="BinaryData"/> instance representing the serialized message.</returns>
-        public virtual BinaryData ToBytes()
+        public BinaryData ToBytes()
```

Comment:
We're not concerned with testing/mocking here?   That seems odd.

Reply:
Well mocking is not needed as there are no service calls. Customers can just create an instance of the message and let the serialization do its thing. Is there some scenario where you expect customers to want to mock?